pkgs/development/lua-modules/overrides.nix

```diff
@@ -477,6 +478,22 @@ with prev;
     ];
   });
 
+  magick = prev.magick.overrideAttrs (oa: {
+    buildInputs = oa.buildInputs ++ [
+      imagemagick
+    ];
+
+    # Fix MagickWand not being found in the pkg-config search path
```

Comment:
have you tried adding pkg-config to nativeBuildInputs ? anyway luarocks doesn't use pkg-config and I dont see the code that would use it (`get_flags` maybe) so unlikely to work.

Reply:
Yeah I tried that earlier and it didn't work